packages/console/src/components/UserName/index.module.scss

```diff
@@ -4,13 +4,23 @@
   font: var(--body-medium);
   color: var(--color-text);
 
-  .userId {
+  .title {
+    display: inline-block;
+    max-width: 100%;
+    overflow: hidden;
+    @include _.text-ellipsis;
```

Comment:
And I think all table columns should have this `text-ellipsis` mixin applied

Reply:
I have tried it before and found it caused some unexpected behaviors. 
e.g.
<img width="349" alt="image" src="https://user-images.githubusercontent.com/10806653/215510630-197f70bc-f936-4a2e-ad43-90e92f3a438f.png">
Also, some data need to be displayed in more than one line.

Reply:
Then these cases are the exceptions, we can use CSS to cancel the ellipsis. What do you think?

Reply:
I think we can not assume data in the table data should use text ellipsis by default.
1. It changes the default style of the table cell
2. Using CSS to cancel the ellipsis requires us to write a lot of styles to cover the ellipsis needs, which will make the code not so intuitive